vassal-app/src/main/java/VASSAL/tools/imports/adc2/ADC2Module.java

```diff
@@ -798,7 +799,7 @@ protected PieceSlot getPieceSlot() {
     private final SymbolSet.SymbolData hiddenSymbol;
     private final int hiddenPieceOptions;
     private final int order;
-    private final TreeSet<Player> allies = new TreeSet<>(Comparator.comparingInt(p -> p.order));
+    private final Set<Player> allies = new TreeSet<>(Comparator.comparingInt(p -> p.order));
```

Comment:
These TreeSet ones should use the OrderedSet interface, since I think the reason TreeSet is being used is to preserver ordering.

Reply:
OrderedSet isn't a thing... SortedSet? NavigableSet?

Reply:
SortedSet